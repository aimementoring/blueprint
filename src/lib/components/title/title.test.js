import React from 'react';
import renderer from 'react-test-renderer';
import Title from './title.js';

describe('Title', () => {
  it('renders properly', () => {
    const tree = renderer.create(<Title>h1 default title</Title>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('is backwards compatible (text passed as prop)', () => {
    const tree = renderer.create(<Title text="h1 default title" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
