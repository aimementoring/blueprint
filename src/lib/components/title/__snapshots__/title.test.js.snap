// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title is backwards compatible (text passed as prop) 1`] = `
<div
  className="theme-base"
>
  <h1
    className="h1Title"
  >
    h1 default title
  </h1>
</div>
`;

exports[`Title renders properly 1`] = `
<div
  className="theme-base"
>
  <h1
    className="h1Title"
  >
    h1 default title
  </h1>
</div>
`;
